src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -66,11 +66,50 @@ Cross identity ID copying is only allowed if the destination path has the the ri
 
 </Callout>
 
-### More `copy` options
+## Specify a bucket or copy across buckets / regions
 
-Option | Type | Description | Reference Links |
-| -- | -- | ----------- | -- |
-| useAccelerateEndpoint | boolean | Whether to use accelerate endpoint. | [Transfer Acceleration](/[platform]/build-a-backend/storage/extend-s3-resources/#example---enable-transfer-acceleration) |
+Alternatively, choose a specific target by including `bucket` in both the source and destination objects. 
+
+See [additional storage bucket usage](/[platform]/build-a-backend/storage/set-up-storage/#additional-storage-bucket-usage) for more information.
```

Comment:
i dont see it as a sub section on the right side navigation. Could you check on that?

Reply:
I'm assuming you mean the link target on the Setup Storage page? In that case, resolved.